app/views/response/response.html.erb

```diff
@@ -1,434 +1,553 @@
-          switch(data[index].qtype){
-            case "Criterion":
-              if(data[index].answer != null) {
-                $('#responses_' + index + '_score').val(data[index].answer);
-            <% if @dropdown_or_scale == "dropdown" %>
-                  $('#responses_' + index + '_score').barrating('set', data[index].answer);
-            <% else %>
-                $("input[name='Radio_" + data[index].question_id +"']").each(function(){
-                  if(data[index].answer == $(this).val())
-                    $(this).prop("checked", "true");
-                  else
-                    $(this).removeProp("checked");
-                });
-            <% end %>
-              }
-              if(data[index].comments != null)
-                tinymce.get('responses_' + index + '_comments').setContent(data[index].comments);
-              break;
-            case "TextArea" :
-              if(data[index].comments != null)
-                tinymce.get('responses_' + index + '_comments').setContent(data[index].comments);
-              break;
-            case "TextField":
-              if(data[index].comments != null)
-                $('#responses_' + index + '_comments').val(data[index].comments);
-              break;
-            case "Checkbox" :
-              if(data[index].answer != null)
-                $('#responses_' + index + '_score').prop( "checked", data[index].score == 1).change();
-              break;
-            case "Dropdown" :
-              if(data[index].comments != null) {
-                $('#responses_' + index + '_comments').val(data[index].comments);
-              }
-              break;
-            case "Scale":
-              $("input[name='Radio_" + data[index].question_id +"']").each(function(){
-                if(data[index].answer == $(this).val())
-                  $(this).prop("checked", "true");
-                else
-                  $(this).removeProp("checked");
-              });
-              break;
-            default:
-          }
-        }
-        // release the form
-        removeOverlay();
-      });
-    }
-
-    // update the form after it's back to the foreground.
-    // save review when the window is sent to background
-    vis(function () {
-      // reload review when the window is sent to the foreground
-      if (vis() && $("#autosave_cbx").prop("checked")){
-        // freeze the form until it's updated
-        displayOverlay("Loading");
-        // delay one second to let the data saved if the user switch back and forth
-        setTimeout(function() {
-          updateForm();
-        }, 1000);
-      }else if (!vis() && $("#autosave_cbx").prop("checked")){
-        executeSave();
-        //console.log("Saving before going to background at " + new Date().toLocaleString('en-US', { weekday: 'short', day: 'numeric', month: 'short', year: 'numeric', hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: true }));
```

Comment:
Leaving this in created a --verbose error in Travis CI and was therefore removed.